pkg/cloudprovider/aws/apis/v1alpha1/register.go

```diff
@@ -32,6 +32,7 @@ var (
 	AWSRestrictedLabelDomains = []string{
 		"k8s.aws",
 	}
+	OperatingSystemBottleRocket = "bottlerocket"
```

Comment:
Kubernetes convention for yaml values is UpperCamelCase or `BottleRocket` `EKSOptimized`